.github/workflows/python-package.yml

```
@@ -18,7 +18,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11', '3.12']
+        python-version: ['3.8', '3.9', '3.10', '3.11', '3.12', '3.13']
```

```suggestion
        python-version: ['3.9', '3.10', '3.11', '3.12', '3.13']
```
Since you already removed 3.7 as end-of-life: 3.8 is EOL too https://endoflife.date/python